### DISCUSSION TITLE
How to create a new meter on self hosted openmeter?

### DISCUSSION THREAD

**Discussion by Mohit21GoJs** [openmeterio/openmeter#1850] [Q&A]
Hi Team, 

We are using self hosted openmeter version and would like to create additional meters on openmeter, however the [documentation](https://openmeter.io/docs/api/open-source) do not list any endpoints to create meters, could you please help us understand how to setup custom meters in our self hosted openmeter service?

**Comment by SelvamArul** [comment]
@Mohit21GoJs Meters are defined in the [config.yaml](https://github.com/openmeterio/openmeter/blob/main/quickstart/config.yaml) file, which the individual services defined in [docker-compose.yaml](https://github.com/openmeterio/openmeter/blob/main/quickstart/docker-compose.yaml) refer to.
 e.g. [here](https://github.com/openmeterio/openmeter/blob/46154588472085e2b40506be1fafff86f4efb77f/quickstart/docker-compose.yaml#L4) & [here](https://github.com/openmeterio/openmeter/blob/46154588472085e2b40506be1fafff86f4efb77f/quickstart/docker-compose.yaml#L17C10-L17C48).

**Reply by DoonPort0422** [reply]
Can apply the update of config.yaml, without restart container?
